### ISSUE TITLE
Sometimes I get this error, offsetTop

### ISSUE THREAD

**Issue by ikkysleepy** [imzbf/md-editor-v3#283]
When I select the top image I get this error:

```
Cannot read properties of undefined (reading 'offsetTop')
TypeError: Cannot read properties of undefined (reading 'offsetTop')
    at HTMLDivElement.u (webpack-internal:///./node_modules/md-editor-v3/lib/es/MdEditor.mjs:1160:37)
```

the top image is this:
```
![](https://127.0.0.1:8443/photos/6c62596e6376514e4c4376416a472f65646c4d4d6a4870595a722f62)
```

 I seen this error a couple of times.

**Comment by imzbf** [comment]
I got this issue. Thanks

**Comment by imzbf** [comment]
I fixed it in version `4.0.4`, have a try.
